packages/karamel/karamel.1.0.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+maintainer: "protz@microsoft.com"
+authors: "Jonathan Protzenko <jonathan.protzenko@gmail.com>"
+homepage: "https://github.com/fstarlang/karamel"
+license: "Apache-2.0"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "ocamlfind" {build}
+  "batteries"
+  "zarith"
+  "stdint"
+  "yojson"
+  "ocamlbuild" {build}
+  "fileutils"
+  "menhir" {>= "20161115"}
+  "pprint"
+  "ulex"
+  "process"
+  "fix"
+  "visitors"
+  "wasm" {= "1.1.1"}
+  "ppx_deriving"
+  "ppx_deriving_yojson"
+  "fstar" {= "2022.01.15"}
+]
+depexts: ["coreutils"] {os = "macos" & os-distribution = "homebrew"}
+conflicts: "kremlin" {!= "transition"}
```

Comment:
this is confusing. transition package usually simply depend on the new package for easy transition but here this makes it impossible to do. What’s the purpose of the transition package then?

Reply:
nevermind i read it backward, now that the transition package depends on the right package it's perfectly fine. Sorry for the dumb comment.

Reply:
Apologies, I might have misunderstood the documentation. Thanks for the help and for merging!

Reply:
Sorry if something was badly explained. If you think something can be done better feel free to open a PR to modify this documentation: https://github.com/ocaml/opam/blame/master/doc/pages/Tricks.md